docs/tokens/esdt-tokens.md

```diff
@@ -624,6 +624,30 @@ RolesAssigningTransaction {
 
 _For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
 
+### **Managing Token Transferability**
+
+The manager of an ESDT token can restrict transferability by setting **ESDTTransferRole** to specific addresses. 
+
+When no address has the role, the token can be transferred freely. If at least one address has the role, only addresses with the role can send or receive the token. An address without the role can still transfer tokens to an address with the role.
```

Comment:
>If at least one address has the role, only addresses with the role can send or receive the token. An address without the role can still transfer tokens to an address with the role.

a bit contradictory. "only addresses with the role can send or receive" vs "An address without the role can still transfer"

Reply:
edited